test/python/circuit/test_controlled_gate.py

```diff
@@ -887,6 +887,22 @@ def test_open_controlled_equality(self):
             XGate().control(1, ctrl_state='0'),
             XGate().control(1, ctrl_state='1'))
 
+    def test_cx_global_phase(self):
+        """
+        Test controlling CX with global phase
+        """
+        theta = pi/2
+        circ = QuantumCircuit(2, phase=theta)
+        circ.cx(0, 1)
+        cx = circ.to_gate()
+        self.assertNotEqual(Operator(CXGate()), Operator(cx))
+
+        ccx = cx.control(1)
+        base_mat = Operator(cx).data
+        target = _compute_control_matrix(base_mat, 1)
+
+        self.assertEqual(Operator(ccx), Operator(target))
```

Comment:
This test should be updated to test that the result is a CCX with a single additional U1 on the control wire. (more explanation in the summary).

Reply:
done